==== //sw/main/apps/OpenGL/glut/lib/glut/win32_x11.c#6 - d:\src\sw\main\apps\OpenGL\glut\lib\glut\win32_x11.c ====
***************
*** 6,13 ****
     and is provided without guarantee or warrantee expressed or 
     implied. This program is -not- in the public domain. */
  
! #include <stdio.h>
! #include "win32_x11.h"
  
  /* global variable that must be set for some functions to operate
     correctly. */
--- 6,12 ----
     and is provided without guarantee or warrantee expressed or 
     implied. This program is -not- in the public domain. */
  
! #include "glutint.h"
  
  /* global variable that must be set for some functions to operate
     correctly. */
==== //sw/main/apps/OpenGL/glut/lib/glut/win32_glx.c#8 - d:\src\sw\main\apps\OpenGL\glut\lib\glut\win32_glx.c ====
***************
*** 5,12 ****
     and is provided without guarantee or warrantee expressed or 
     implied. This program is -not- in the public domain. */
  
! #include <stdio.h>
! #include "win32_glx.h"
  
  /* global current HDC */
  extern HDC XHDC;
--- 5,11 ----
     and is provided without guarantee or warrantee expressed or 
     implied. This program is -not- in the public domain. */
  
! #include "glutint.h"
  
  /* global current HDC */
  extern HDC XHDC;
